README.adoc

```diff
@@ -50,7 +53,11 @@ The application that you will work with is a `query` service, which fetches info
 include::{common-includes}/gitclone.adoc[]
 
 // =================================================================================================
-== Running the application in the testing environment
+== Trying the basic configuration
+
+Run the `query` service in development mode with the `system` service that is running in the testing enivronment.
```

Comment:
you also might revise this sentence so it reflects the order of the tasks in this section, like

```suggestion
Deploy the `system` service in the testing environment and access it with the `query` service, which is running in dev mode.
```